config/detekt/detekt.yml

```diff
@@ -426,13 +426,14 @@ naming:
 #    forbiddenName: []
   FunctionMaxLength:
     active: true
-    maximumFunctionNameLength: 50
+    maximumFunctionNameLength: 120
   FunctionMinLength:
     active: true
     minimumFunctionNameLength: 3
   FunctionNaming:
     active: true
-    excludes: ['**/test/**', '**/androidTest/**', '**/commonTest/**', '**/jvmTest/**', '**/jsTest/**', '**/iosTest/**']
+    excludes: ['**/test/**', '**/integrationTest/**']
+    #functionPattern: '[a-z`][a-zA-Z0-9`\s]*'
```

Comment:
Do we need to increase allowed function name length if we exclude integration test files from that check?

Reply:
I have disabled the `FunctionMaxLength` check, as it does not provide any options to exclude test sources.